test/test_modifiers.py

```diff
@@ -0,0 +1,62 @@
+"""
+Tests brewblox_codec_spark.modifiers
+"""
+
+import pytest
+from brewblox_codec_spark import modifiers
+from brewblox_codec_spark.proto import OneWireTempSensor_pb2
+
+
+@pytest.fixture(scope='module')
+def mod():
+    return modifiers.Modifier('config/fahrenheit_system.txt')
+
+
+def encode_temp_sensor_data():
+    return {
+        'settings': {
+            'address': 'address',
+            'offset[delta_degF]': 20
+        }
+    }
+
+
+def decode_temp_sensor_data():
```

Comment:
rename encode_temp_sensor_test_input? I read encode as a verb, like 'this function encodes something'. I can see it doesn't, but the name could be clearer.